packages/cdc-react/src/components/ProfileHeader/ProfileHeaderPopupMenuItem/ProfileHeaderPopupMenuItem.tsx

```diff
@@ -25,6 +25,7 @@ export const ProfileHeaderPopupMenuItem = ({
 }: ProfileHeaderPopupMenuItemProps & JSX.IntrinsicElements["button"]) => {
   return (
     <button
+      tabIndex={1}
```

Comment:
Why always 1?

Reply:
It's a grouping of tabIndexes I thought. So all 1's tab together.